content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -229,7 +229,7 @@ constructor, or just one method:
 [source,groovy]
 ----
 package org.foo
-class Utilities {
+class Utilities implements Serializable {
```

Comment:
Does this need updating as well? https://github.com/jenkins-infra/jenkins.io/pull/495/files#diff-d0909a306d46ea261898f36947fb9d9bL260